source/_integrations/compensation.markdown

```diff
@@ -0,0 +1,66 @@
+---
+title: Compensation
+description: Instructions on how to integrate compensation sensors into Home Assistant.
+ha_category:
+  - Utility
+  - Sensor
+ha_iot_class: Local Push
+ha_release: '0.117'
+ha_quality_scale: internal
+ha_codeowners:
+  - '@petro31'
+ha_domain: compensation
+---
+
+The `compensation` sensor platform consumes the state from other sensors. It exports the `compensated` value as state and the following values as attributes: `entity_id` and `coefficients`.
+
+## Configuration
+
+To enable the compensation sensor, add the following lines to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: compensation
```

Comment:
We no longer accept YAML configuration for integrations on a platform level.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>

Reply:
Ok, I'll probably need to update the other PR to handle that.

Reply:
fixed